ios/RCTWebRTC/WebRTCModule+RTCMediaStream.m

```diff
@@ -232,7 +282,7 @@ - (RTCVideoTrack *)createVideoTrack:(NSDictionary *)constraints {
   RTCMediaStreamTrack *track = self.localTracks[trackID];
   if (track) {
     RTCVideoTrack *videoTrack = (RTCVideoTrack *)track;
-    [videoTrack.videoCaptureController switchCamera];
+    [(VideoCaptureController *)videoTrack.captureController switchCamera];
```

Comment:
Check the instance type here, since it could be called on a screen sharing stream.

Reply:
reverted